gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -82,6 +85,10 @@ def get_thruster_wrench(self, position, direction):
         wrench_column = np.hstack([forces, torques])
         return np.transpose(wrench_column)
 
+    def get_b_matrix(self, srv):
```

Comment:
Hm yeah, it probably should be thread locked, and the "thruster_failure" alarm handler should alert the TGEN that it needs to request a new B as soon as the thruster mapper is done re-computing it.